### PR TITLE
Completed css-fonts

### PR DESCRIPTION
**Pull request by julieluangpraxay**
Screenshot 2023-07-19 at 4 22 17 PM

### CODE REVIEW

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+h1 {
+  font-family: 'Bebas Neue', sans-serif;
+  line-height: 150%;
+}
+
+h2,
```
Comment (shawnkost) [inline css-fonts/styles.css:6]: Your `h1` & `h2` should be using the same `font-family`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Julie!

File: css-fonts/styles.css (hunk 1 above)
Comment (julieluangpraxay) [reply css-fonts/styles.css]: Screenshot 2023-07-19 at 4 41 13 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**